File: power/power.py
def power(x,n):
  orig_n = None
  c = 1
  r = x
  if n == 0:
    return 1
  if n < 0:
    orig_n = n
    n = orig_n + (-orig_n) + (-orig_n)
  while c < n:
    r = r * x
    c += 1
  if orig_n is None:
    return r
  else:
    r1 = 1 / float(r)
    return r1

File: Math/power-x-n/power.cpp
#include<iostream.h>

//function to calculate the nth power of x 
float power(int a,int b)
{
  float pro=1;
  for(int i=1;i<=n;i++)
    pro=pro*x;  
  return pro;    
}
void main()
{ 
  float pow = power(x,n);
  std::cout<<x<<" raised to the power "<<n<<" is: "<<pow;
  
  /*
  Exapmles:
  power(2,10)=1024
  power(3,8)=6561
  */
}
